README.md

```diff
@@ -270,6 +270,7 @@ A collection of awesome browser-side [JavaScript](https://developer.mozilla.org/
 * [The JavaScript that you should know](https://medium.com/@pedropolisenso/o-javasscript-que-você-deveria-conhecer-b70e94d1d706) - Article about concepts of JavaScript Functional.
 * [How JavaScript works](https://blog.sessionstack.com/tagged/tutorial) - A series of articles about the building blocks of JavaScript.
 * [Multi-threading using web-workers](https://www.loginradius.com/engineering/blog/adding-multi-threading-to-javascript-using-web-workers/) - Web Workers: Adding Multi-threading to JavaScript
+* [this keyword in Javascript](https://www.loginradius.com/engineering/blog/breaking-down-this-keyword-in-javascript/) - Breaking down the 'this' keyword in Javascript
```

Comment:
It's `JavaScript`. Everytime someone writes `Javascript` or `javascript` a kitten dies.

Reply:
I committed a crime and you saved me before public got to know about it. Updated it to JavaScript.  :P